myosuite/envs/myo/myochallenge/run_track_v0.py

```diff
@@ -506,7 +517,7 @@ def get_internal_qpos(self):
         to_remove = [self.sim.model.joint('osl_knee_angle_r').qposadr[0].copy(), self.sim.model.joint('osl_ankle_angle_r').qposadr[0].copy()]
 
         temp_qpos[to_remove] = 100
-        temp_qpos[temp_qpos != 100]
+        temp_qpos = temp_qpos[temp_qpos != 100]
```

Comment:
why is this condition needed?

Reply:
All the OSL joint observations need to be removed. This is just to filter and shorten the final array

Reply:
would it be possible to add the number of physiological joints as variable instead of hardcode the number?

Reply:
Yes. I can do that. Give me a moment

Reply:
Ok done. I have also done the same for the muscles as well.